2.6/alpine/Dockerfile

```diff
@@ -53,4 +57,4 @@ EXPOSE 2019
 
 WORKDIR /srv
 
-CMD ["caddy", "run", "--config", "/etc/caddy/Caddyfile", "--adapter", "caddyfile"]
+CMD ["caddy", "run", "--config", "/etc/caddy/Caddyfile", "--adapter", "caddyfile"]
```

Comment:
I see a missing newline as the only change - I suspect your editor is misconfigured... Can you restore this? (same in the other files in this PR)

Reply:
Actually, there is an .editorconfig file in this repo, and these Dockerfiles do not adhere to its rules, but my editor does so I disabled some rules to keep my changes minimal, but I seem to have missed one.

Reply:
The rules say that files with extension .tmpl should have `insert_final_newline = false`, which results in the Dockerfiles also having that once generated, even though the .editorconfig would otherwise ensure those had a newline at the end.

Reply:
Let me know how you think we should proceed and I'll happily oblige, but it's hard to know exactly how you guys want styling considering you're not obeying your own configurations, and are using different styling in different templates but apparently explicitly prefer one over the other...

Reply:
Sorry about that 😓 don't mean to cause friction!

Thanks for all the help, it's very much appreciated.

I'll look into making a PR for xcaddy today (hopefully).

Reply:
Haha it's no problem guys, just pointing it out 😛

Reply:
@hairyhenderson How do you think we should proceed? Should I update the .editorconfig according to your expectations? Should I "normalise" the repo to adhere to the .editorconfig? Should we delete the .editorconfig?

Reply:
sorry, I missed these responses... let's delete the `.editorconfig` - IMO it's not useful in this repo.

Reply:
ok, done in 6fa9eafebf128071ebdd129b459ecd8e960bd470